### ISSUE TITLE
Performance on QNLI dataset?

### ISSUE THREAD

**Issue by ZhengtianXu** [namisan/mt-dnn#34]
On the glue leaderboard, the QNLI performance can get 96.0. But when I train the network, the dev accuracy can only get 92.2. I don't have V100 GPU. The training setting is batch size 8 on 8 TITAN X. Can you provide some details when you train the network on QNLI dataset?

**Comment by devesh-tewari** [comment]
I have been working on QnA tasks. Can you please upload the fine tuned model for QNLI? I don't have a good GPU.

**Comment by namisan** [comment]
Please refer our new paper for the details, e.g., caption of Tab 2.
https://arxiv.org/pdf/1904.09482.pdf

We'll consider to upload QNLI models.
